### PR TITLE
Implement notifications for addRecipient and removeRecipient hooks

### PR DESCRIPTION
**Pull request by dmolineus**
This pull requests provides notifications for the `addRecipient` and `removeRecipient` hook. It introduces a new notification type `newsletter` which is used for both hooks.

This PR does not implement notifications instead of the default mails (See #150).

/cc @frontendschlampe

### CODE REVIEW

File: config/runonce.php [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (dmolineus) [reply config/runonce.php]: Restored.

**Comment by frontendschlampe** [comment]
yes, we will test it. We need it for a project! :-)

**Comment by Toflar** [comment]
Okay, go ahead then 😄 There's `dev-release/1.5.0` available now. Also maybe give https://github.com/terminal42/contao-notification_center/issues/157 a try 😄

**Comment by aschempp** [comment]
As discussed with @frontendschlampe I have reverted the changes. Please create a new pull request with the updated feature.

**Pull request merged**